docs/reference-guides/data/data-core-block-editor.md

```diff
@@ -477,6 +477,10 @@ _Returns_
 
 -   `Array`: ids of top-level and descendant blocks.
 
+### getDefaultInspectorControlsTab
```

Comment:
As it's likely that these will be superceeded by a dedicated package or mechanic for interacting with the UI (as suggested by @talldan) should we mark as `unstable`?